capsules/src/net/icmpv4/icmpv4.rs

```diff
@@ -0,0 +1,169 @@
+//! ICMP layer that works with IPv4.
```

Comment:
Why have this at all? Is it meaningful to send ICMPv4 packets over an IPv6 link?